### DISCUSSION TITLE
How to link to a project

### DISCUSSION THREAD

**Discussion by LeonardoGentile** [marcusolsson/obsidian-projects#793] [Help]
Hello,
as I've understood the various projects are not saved in a markdown file.
So my questions is, how can I open the project view (of a specific project) by pointing at it via a wiki link like [[My Project Name]]?

Thanks

**Comment by tosascott** [comment]
I too would like to be able to open a specific vault, project & view using a link. @LeonardoGentile, I don't see a clean way to override internal links syntax ala `[[Obsidian Kanban]]`, but perhaps the [Obsidian URL protocol](https://help.obsidian.md/Extending+Obsidian/Obsidian+URI) could be used. For example, `obsidian://projects?project=Obsidian&view=Board`.

If extending the URL protocol is outside the Projects plugin design, then [Obsidian Advanced URI](https://vinzent03.github.io/obsidian-advanced-uri/) can be used to map a URL to the command palette. For example, to open the **Projects** tab in my _Personal_ vault I can use: `obsidian://advanced-uri?vault=Personal&commandid=obsidian-projects%3Ashow-projects`

Now I just came across **Obsidian Advanced URI** yesterday, and I'm a novice at the Obsidian API, but I don't see a way to override the defaults and set the preferred project and preferred view using the command palette without more Projects plugin support. If there was parameter support, then the URL would look like `obsidian://advanced-uri?vault=Personal&commandid=obsidian-projects%3Ashow-projects&project=Outdoors&view=Board`

**Comment by jebeaudoin** [comment]
The answer is [here in #377](https://github.com/marcusolsson/obsidian-projects/discussions/377#discussion-4772762). It requires the Obsidian Advanced URI plugin. Basically, if you enable the Projects plugin setting to show your specific project, you can grab a specific URI and set up a wikilink to link to it from a note.
